front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/index.tsx

```diff
@@ -123,15 +147,38 @@ const InputsTable = ({
             <colgroup>
               <col span={1} style={{ width: '5%' }} />
               <col span={1} style={{ width: '30%' }} />
-              <col span={1} style={{ width: '65%' }} />
+              {query.category && <col span={1} style={{ width: '20%' }} />}
+              <col span={1} style={{ width: query.category ? '45%' : '65%' }} />
             </colgroup>
             <thead>
               <tr>
                 <th>
                   <Checkbox checked={false} onChange={handleCheckboxChange} />
                 </th>
                 <th>{formatMessage(messages.inputsTableInputs)}</th>
-                <th>{formatMessage(messages.inputsTableCategories)}</th>
+                <th>
+                  {query.category ? (
+                    <StyledSort
+                      onClick={onSort}
+                      as="button"
+                      data-testid="insightsSortButton"
+                    >
+                      {formatMessage(messages.inputsTableCategories)}
+                      <Icon
+                        name={
+                          query.sort === '-approval'
+                            ? 'chevron-up'
+                            : 'chevron-down'
+                        }
+                      />
+                    </StyledSort>
+                  ) : (
+                    formatMessage(messages.inputsTableCategories)
```

Comment:
I actully like that you use formatmessage instead of FormattedMessage, Less domnodes (given the version of react-intl we're working with)

Reply:
Yes, I do prefer it, it makes it feel a bit cleaner 👍